backend/src/services/interview/interview.service.ts

```diff
@@ -143,18 +145,34 @@ class Interview {
 
   public async update(data: {
     id: number;
+    user: UserWithPermissions;
     interviewUpdateInfoRequestDto: InterviewsUpdateRequestDto;
   }): Promise<InterviewsByIdResponseDto> {
-    const { id, interviewUpdateInfoRequestDto } = data;
+    const { id, interviewUpdateInfoRequestDto, user } = data;
     const { interviewerUserId, status, interviewDate } =
       interviewUpdateInfoRequestDto;
 
-    const interview = await this.#interviewRepository.update({
-      id,
-      interviewerUserId,
-      status,
-      interviewDate,
+    const hasInterviewsPermission = checkHasPermission({
+      permissionKeys: [PermissionKey.MANAGE_INTERVIEWS],
+      userPermissions: user.permissions,
     });
+    const hasInterviewerUserId = interviewerUserId !== undefined;
```

Comment:
```suggestion
    const hasInterviewerUserId = Boolean(interviewerUserId);
```

Reply:
Then if we set null value for interviewerUserId throught Api the check will be passed

Reply:
Then maybe it's better to check like
```suggestion
    const hasInterviewerUserId = 'interviewerUserId' in interviewUpdateInfoRequestDto;
```